submit_cgap/submission.py

```diff
@@ -59,7 +58,7 @@ def resolve_server(server, env):
     Given a server spec or a beanstalk environment (or neither, but not both), returns a server spec.
 
     :param server: a server spec or None
-      A server is the first part of a url (containing the schema, host and, optionally, port).
+      A server is the first part of an url (containing the schema, host and, optionally, port).
```

Comment:
Can we compromise on "a URL"? I don't pronounce this the same as you, but maybe you would say "a" if it was in caps? :)